static/app-strings.json

```diff
@@ -2246,5 +2246,11 @@
   "Attach images by pasting or drag-and-drop.": "Attach images by pasting or drag-and-drop.",
   "There was a network error, but the publish may have been completed. Wait a few minutes, then check your Uploads or Wallet page to confirm.": "There was a network error, but the publish may have been completed. Wait a few minutes, then check your Uploads or Wallet page to confirm.",
   "Sports": "Sports",
-  "--end--": "--end--"
+  "--end--": "--end--",
```

Comment:
`--end--` needs to be at the end.  
- There are still logic that depends on it being that way indirectly (can't recall where, only remember that it's not obvious)
- lt also reduces an unnecessary diff of adding and removing the last comma in the json